homeassistant/components/sensor/serial_pm.py

```diff
@@ -21,7 +21,6 @@
 CONF_BRAND = "brand"
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
-    vol.Required(CONF_PLATFORM): 'serial_pm',
```

Comment:
I'm also confused. It worked here before I installed 0.26
With the 0.26 release, I get the following error message: 
ERROR:homeassistant.bootstrap:Invalid config for [sensor.serial_pm]: required key not provided @ data['platform']. Got None

When I removed the CONF_PLATFORM as a required parameter, it loads the sensor (platform is still given in the configuration)